custom_components/daikinskyport/daikinskyport.py

```diff
@@ -3,12 +3,15 @@
 import json
 import os
 import logging
+import aiohttp
 
-from requests.exceptions import RequestException
+import async_timeout
+
+from requests.exceptions import RequestException, ConnectionError, Timeout
 
 from .const import DAIKIN_PERCENT_MULTIPLIER
 
-logger = logging.getLogger('daikinskyport')
+_LOGGER = logging.getLogger('daikinskyport')
```

Comment:
Hi @joejulian,

I'm not very familiar with HA internals or proper coding conventions for HA components. Can you share why you changed the name of the logging object?

I also wonder, if it's not directly related to config flow, if this should be split to a separate PR.

Reply:
I'm learning. Comparing what's here with newer examples, I'm not yet sure what's needed and what's not. I figured I'd open this to get feedback and once I have a handle on what's going on, I can close it and split it up into single topic PRs.